.github/codecov.yml

```diff
@@ -23,7 +23,32 @@ comment:
   require_changes: no
 
 ignore:
-  - "docs/**/*"
-  - "examples/**/*"
-  - "test/**/*"
   - "**.md"
+  - ".github"
+  - ".vscode"
+  - "build"
+  - "docs"
+  - "examples"
+  - "src/Shared"
+  - "test"
+
+flags:
+  unittests-Solution-Stable:
+    carryforward: true
+    paths:
+      - src
+
+  unittests-Solution-Experimental:
+    carryforward: true
+    paths:
+      - src
+
+  unittests-Instrumentation-Stable:
+    carryforward: true
+    paths:
+      - src
```

Comment:
Update path for instrumentation?

Reply:
Not sure on this either. [InstrumentationLibraries.proj](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/InstrumentationLibraries.proj) actually builds and tests everything so this _should_ be fine but 🤷 I'm considering code coverage as a WIP!